src/ir/module.rs

```diff
@@ -19,6 +20,7 @@ pub struct Module<'a> {
     pub start_func: Option<Func>,
     pub debug: Debug,
     pub debug_map: DebugMap,
+    pub custom_sections: IndexMap<String,Vec<u8>>
```

Comment:
Would you mind running `cargo fmt` on your changes? This appears not to be rustfmt'd...